src/clj/vanilla/db/core.clj

```diff
@@ -474,4 +511,66 @@
   (conj (:keys (first data-2)) "selected" "sliced")
   (map #(conj % false (< (second %) slice-at)) (:data (first data-2)))
 
+
+
+
+  ())
+
+(comment
+
+  (sql/query vanilla-db ["SELECT * FROM services"])
+  (sql/query vanilla-db ["SELECT * FROM users"])
+  (sql/query vanilla-db ["SELECT * FROM layout"])
+  (sql/query vanilla-sqlite ["SELECT * FROM services"])
+  (sql/query vanilla-sqlite ["SELECT * FROM users"])
+  (sql/query vanilla-sqlite ["SELECT * FROM layout"])
+
+  (def postgres-data
+    [#:layout{:ret_types "[:data-format/x-y :data-format/x-y-n :data-format/x-y-e :data-format/y]",
+              :name ":area-widget",
+              :username "chad",
+              :basis ":chart",
+              :data_grid "{:x 3, :y 0, :w 5, :h 15}",
+              :type ":area-chart",
```

Comment:
Maybe clean up this large comment to just the postgres one, and just one set of layout data for reference

Reply:
I'll have to do another pass and clean out old data that's not used anymore. Rich comments that pertain to legacy/cleaned out code help no one haha